rotkehlchen/tests/unit/decoders/test_cowswap.py

```diff
@@ -63,7 +63,7 @@ def test_swap_token_to_token(ethereum_inquirer, ethereum_accounts):
             asset=Asset('eip155:1/erc20:0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48'),
             balance=Balance(amount=FVal('3800')),
             location_label=user_address,
-            notes='Receive 3800 USDC as the result of a swap in cowswap',
+            notes='Receive 3800 USDC as the result of a cowswap market order swap',
```

Comment:
Change the decoder to fix the wording here. Market order is enough, swap as wording is not needed

```suggestion
            notes='Receive 3800 USDC as the result of a cowswap market order',
```